src/main/java/frc/robot/RobotContainer.java

```diff
@@ -114,6 +122,23 @@ private void configureBindings() {
     conOperator.btn_A
         .onTrue(
             Commands.runOnce(() -> subCollector.setPivotMotorAngle(prefCollector.climbPivotAngle.getValue())));
+
+    // Spin Charger treads
+    conOperator.btn_RBump
+        .onTrue(Commands.runOnce(() -> subChargerTreads.setMotorSpeed(prefChargerTreads.motorSpeed.getValue())))
+        .onFalse(Commands.runOnce(() -> subChargerTreads.setMotorSpeed(0)));
+
+    // Rotate drivetrain wheels in charge station orientation
+    conOperator.POV_North
```

Comment:
You said these commands don't work; please remove them + their preferences if they don't

Reply:
Referenced in #135